pkg/apis/vpc/v1alpha1/subnetport_types.go

```diff
@@ -16,6 +16,8 @@ type SubnetPortSpec struct {
 	SubnetSet string `json:"subnetSet,omitempty"`
 	// AddressBindings defines static address bindings used for the SubnetPort.
 	AddressBindings []PortAddressBinding `json:"addressBindings,omitempty"`
+	// VIF attachment of the SubnetPort.
+	Attachment PortAttachment `json:"attachment,omitempty"`
```

Comment:
Does PortAttachment match the field in SubnetPort spec in NSX API?
 
Today, who generates VIF ID? NSX or nsx-op?

Reply:
Yes，this match the field in the NSX API. Today nsx-op auto-generates the VIF ID.

Reply:
We synced with Yanlei this noon and they do not need specify VIF ID, so removed this change.